docs/content-guidelines.md

```diff
@@ -0,0 +1,71 @@
+# Kind Content guidelines
+
+We'd like to clarify that these guidelines are only effective on the
+UserStyles.world website.
+
+
+## Purpose
+
+UserStyles.world("USw") encourages their users from anyone who wishes to
+add/import/review/explore UserCSS styles("styles"), to be able to do so.
+People shouldn't feel discouraged from participating in this community.
+
+In order to ensure the safety and quality of styles, we encourage anyone
+to make a conscious effort into following these guidelines
+best to their ability.
+
+We'd like to note and clarify that measurements[¹](#footnote) can be taken
+to ensure safety and quality of the USw platform.
+
+These guidelines suggest specific ways to accomplish that goal.
+
+
+## Guidelines
+
+- As a platform that host UserCSS styles, USw highly discourage to host/post
+    other content then UserCSS styles. 
+
+- USw allow to host screenshot(s) for your style, please try to reflect this 
+    screenshot to show how your style looks like.
+
+- As noted, USw hosts screenshot(s) for their styles, please remember that not
+    everyone is of consent age in their respective country, therefor
+    we discourage the usage of not safe for work("NSFW") pictures
+    for these screenshot(s).
+
+- Please, while you're writing a review of a style to give
+    constructive feedback, this will help the author improve the style.
+    USw discourage to write reviews when these are based on feedback
+    that doesn't help the author.
+    
+- Please, remember that you don't have to write
+    a review if you don't want/have to.
+
+- Please, recognize that criticism/feedback of your styles is not a personal attack
+    on you.
+
+- Please, don't raise unrelated topics/off-topic while writing a review.
+
+- Please, don't use the review/feedback system to report bugs, while these help
+    the author. This system is not the correct way to report them. Try to to
+    contact the author and check for a supportURL in the UserCSS style.
+
+- Please, try to avoid offend on gender, race, ethnic group,
+    physical appearance, religion, cultural background, and any other
+    demographic characteristics views
+    to be reflected on UserCSS Styles, USw highly discourages these styles,
+    as they can offend people in that group.
+    Supporting any of those views is fine to be reflected on UserCSS styles,
+    as long as they don't offend that view.
```

Comment:
Kinda confusing. Try tidying it up.

Reply:
Heh, I couldn't. If you get what I'm trying to say. Please try to improve it.

Reply:
So let's take 2 examples, which both should be covered by this guideline.

Because of 🏳️‍⚧️ 🏳️‍🌈 , which is covered under gender. We are taking it as a example.

1) Let's say Style 1 is going to color youtube.com into a more pride flag-ish background-color. That is perfectly fine because that would support the view.

2) Let's say Style 2 is going to try to undermine the pride and color certain flags like poop color and or things you can imagine which clearly indicate to offend that view.

Reply:
The "view" is what makes it confusing. This is essentially saying to avoid making styles that would ridicule, paint in a negative light, or attack people based on things that they're a part of.

Reply:
Question: why should userstyles.world care about politcal views? I think the best thing to do is to not open that can of worms imo.

Reply:
I'm not big on politics, but we need to set a boundary on what's acceptable and what isn't. Hosting hate userstyles on our site is not something that I would like to see.

Reply:
Fair enough, but I'd change the wording to "Please don't upload styles that discriminate against certain groups of people." - I think that's much more compact and clear, doesn't leave anything important out (imo) and still avoids taking a stance on any specific issue.

Reply:
I agree, it sounds great.

Reply:
Thank you for your suggestion, seems to perfectly word it what was trying to describe. It has been applied.